src/OpenLoco/Objects/DockObject.h

```diff
@@ -29,8 +29,8 @@ namespace OpenLoco
         uint8_t num_aux_02_ent; // 0x13
```

Comment:
looks like I missed these 2 in the rename, but they become somewhat ugly - `numAux01` and `numAux02Ent`?